nntrainer/models/neuralnet.h

```diff
@@ -518,6 +482,17 @@ class NeuralNetwork : public ml::train::Model {
   void disableDynamicFineTuning() { dynamic_training_opt.disable(); }
 
 private:
+  using FlexiblePropTypes =
+    std::tuple<props::Epochs, props::BatchSize, props::SavePath>;
+  using RigidPropTypes = std::tuple<props::LossType>;
+
+  RigidPropTypes model_props;         /**< model props */
+  FlexiblePropTypes model_flex_props; /**< model train props */
+  props::ContinueTrain
```

Comment:
Is there any reason that ContinueTrain is not included in FlexiblePropTypes?

Reply:
No, it has to be part of model_flex_props, I'll fix

Reply:
Updated :) Nice catch!